src/dwarf.cpp

```diff
@@ -803,15 +824,15 @@ die dwarf::implementation::abbreviation_to_die(std::size_t die_address, std::uin
 
     if (abbrev_code == 0) return result;
 
-    result._abbrev_code = abbrev_code;
-
     auto& a = find_abbreviation(abbrev_code);
 
     result._tag = a._tag;
     result._has_children = a._has_children;
+    result._attributes_size = a._attributes.size();
+    result._attributes = alloc_attributes(result._attributes_size);
```

Comment:
Here's where we set the size of the attribute vector, and grab a handful of attribute instances from the allocation pool.